includes.php

```diff
@@ -66,6 +66,191 @@
 		</header>
 		<main>';
 
+$mysqli = new mysqli( 'localhost', 'patchdemo', 'patchdemo', 'patchdemo' );
+if ( $mysqli->connect_error ) {
+	die( $mysqli->connect_error );
+}
+
+function insert_wiki_data( string $wiki, string $creator, int $created, array $patches = [] ) {
```

Comment:
`$created` is never used, which presumably causes the dates to be lost in the `migrateToSql.php` script.

Reply:
Fixed